eodLog/service.go

```
@@ -103,32 +110,34 @@ func openFile(targetFile string) *os.File {
 	return f
 }
 
-func dateConvert(date string) string {
-	dtstr1 := date
-	dt, _ := time.Parse("060102 030405", dtstr1)
-	dtstr2 := dt.Format("02/01/2006")
-	return dtstr2
+func dateTimeConvert(date, logTime string) (string, string) {
+
+	d, _ := time.Parse("2/01/2006", date)
```

Do not ignore errors - rather return the error and have HAL send a message on the botlove group.